ksqldb-common/src/main/java/io/confluent/ksql/serde/WindowInfo.java

```diff
@@ -73,12 +85,14 @@ public boolean equals(final Object o) {
       return false;
     }
     final WindowInfo that = (WindowInfo) o;
+    // we omit `emitStrategy`
```

Comment:
This is just a "philosophical" choice (saying that conceptually a window info doesn't care about emit strategy), rather than a practical choice, right? As in, nothing breaks if we decide to include emitStrategy in the equals/hashCode methods, does it?

Reply:
Some stuff did break... I originally included it and only remove it later and added the comment (cannot remember from the top of my head what did break though).

Reply:
Interesting! I'd be curious to know what broke, but if it's difficult to find out again then don't worry about it.

Reply:
Could not reproduce it -- maybe something change in the middle. Added it to both `equals` and `hashCode`.